bin/snare

```diff
@@ -136,8 +135,7 @@ if __name__ == '__main__':
     page_group.add_argument("--list-pages", help="list available pages", action='store_true')
     parser.add_argument("--index-page", help="file name of the index page", default='index.html')
     parser.add_argument("--port", help="port to listen on", default='8080')
-    parser.add_argument("--interface", help="interface to bind to")
-    parser.add_argument("--host-ip", help="host ip to bind to", default='localhost')
+    parser.add_argument("--host-ip", help="host ip to bind to", default='127.0.0.1')
```

Comment:
Any problems with localhost?

Reply:
This was done to accommodate this change: https://github.com/mushorg/snare/pull/177/files#diff-a17cac3025400a430dedc6984959b596L195
Not sure if it's actually required.